maskit/masks.py

```diff
@@ -347,6 +347,11 @@ def expanded_parameters(self, changed_parameters: np.ndarray) -> np.ndarray:
 
         :param changed_parameters: Current set of differentiable parameters
         """
+        if (
+            changed_parameters.size == self.parameters.size
+            and len(changed_parameters.shape) > 1
+        ):
+            return changed_parameters
```

Comment:
Not sure if doing it like that is correct. You might want to add a comment to explain the reasoning, or just flatten it so that it is compatible.

Reply:
Works like a charm 👍